### PR TITLE
regex cleaning refactored in `raw_to_text.ipynb`

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies
#65 
…

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

update regex for cleaning text

### CODE REVIEW

**Comment by isabelizimm** [comment]
pre-commit failed on some regex specific things (I did not use `\` as it can become problematic, so some lines are too long, and \d is used for matching digits), thoughts on how to continue?

File: notebooks/01_collect_data/raw_to_text.ipynb [hunk 1]
```diff
@@ -17,7 +17,7 @@
   },
```
Comment (MichaelClifford) [inline notebooks/01_collect_data/raw_to_text.ipynb:17]: <p>Can you add some additional documentation a user understands what each of these update regex are attempting to match?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (MichaelClifford) [inline notebooks/01_collect_data/raw_to_text.ipynb:17]: <p>I don't think you need to print the "subset_mboxes" here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (MichaelClifford) [inline notebooks/01_collect_data/raw_to_text.ipynb:17]: <p>Before the saving cell here can you add a small section comparing one or two outputs from the original data to the new data to provide someone a view on how the data has been cleaned and improved. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

**Comment by MichaelClifford** [comment]
@isabelizimm you can add `#noqa` to the end of lines if there really is no way around a unique case that is failing the precommit

**Comment by isabelizimm** [comment]
> @isabelizimm you can add `#noqa` to the end of lines if there really is no way around a unique case that is failing the precommit

Got it! I updated things accordingly for this PR, but understand I have to yield this power wisely :)

**Pull request merged**